composer.json

```diff
@@ -22,7 +22,7 @@
         "silverstripe/sentry": "*"
     },
     "require": {
-        "php": "^7|^8",
+        "php": "^8.1",
```

Comment:
This module will install OK in packages which require `7` or `8`. Indeed, I have it running in a 5.0.x personal project using `silverstripe/recipe-core ^5` and it works just fine.